beacon_node/beacon_chain/src/block_verification.rs

```diff
@@ -187,12 +180,18 @@ pub enum BlockError {
     /// It's unclear if this block is valid, but it conflicts with finality and shouldn't be
     /// imported.
     NotFinalizedDescendant { block_parent_root: Hash256 },
-    /// Block is already known, no need to re-import.
+    /// Block is already known and valid, no need to re-import.
     ///
     /// ## Peer scoring
     ///
     /// The block is valid and we have already imported a block with this hash.
-    BlockIsAlreadyKnown(Hash256),
+    DuplicateFullyImported(Hash256),
```

Comment:
There's one more ocurrence of `BlockIsAlreadyKnown` here:

https://github.com/sigp/lighthouse/blob/c0b4f01cf37257bab8dab2b4312dcf07eac93572/beacon_node/network/src/sync/network_context.rs#L620

Reply:
good catch, done in 0f7b5a14c3fd5bb648434a1d99b99c20229cc58f